source/matrix_free/matrix_free.inst.in

```diff
@@ -39,6 +39,12 @@ for (deal_II_dimension : DIMENSIONS)
     template void internal::MatrixFreeFunctions::ShapeInfo<float>::reinit
     <deal_II_dimension>(const Quadrature<1> &, const FiniteElement
                         <deal_II_dimension,deal_II_dimension> &, const unsigned int);
+    template void internal::MatrixFreeFunctions::ShapeInfo<VectorizedArray<double> >::reinit
+    <deal_II_dimension>(const Quadrature<1> &, const FiniteElement
```

Comment:
do me a favor here and below, and keep the template argument at the function name. That is,
```
template 
void 
internal::MatrixFreeFunctions::ShapeInfo<VectorizedArray<double> >::
reinit<deal_II_dimension>(const Quadrature<1> &, const FiniteElement
```
or similar.